### PR TITLE
Run across multiple GCE zones

### PR DESCRIPTION
**Pull request by AlCutter**

### CODE REVIEW

File: cloud/google/create_new_cluster.sh [hunk 1]
```diff
@@ -49,7 +62,7 @@ echo "============================================================="
 
 # Set gcloud defaults:
 ${GCLOUD} config set project ${PROJECT}
-${GCLOUD} config set compute/zone ${ZONE}
+${GCLOUD} config unset compute/zone
```
Comment (phad) [inline cloud/google/create_new_cluster.sh]: you did this just above in the workaround

File: cloud/google/util.sh [hunk 2]
```diff
@@ -10,18 +10,21 @@ function Header() {
...
   TYPE=$1
   NAME=$2
-  WANT_STATUS=$3
+  ZONE=$3
+  WANT_STATUS=$4
   STATUS=""
   until [[ "${STATUS}" =~ "${WANT_STATUS}" ]]; do
     sleep 1
-    STATUS=$(${GCLOUD} compute ${TYPE} describe ${NAME} | grep "status:")
+    STATUS=$(${GCLOUD} compute ${TYPE} describe ${NAME} --zone ${ZONE} | grep "status:")
```
Comment (phad) [inline cloud/google/util.sh]: You should perhaps specify the grep you want, eg.  /usr/bin/grep

File: cloud/google/create_new_cluster.sh (hunk 1 above)
Comment (AlCutter) [inline cloud/google/create_new_cluster.sh]: Yeah, I just wanted to make sure there wasn't one set by default coming in here.

File: cloud/google/util.sh (hunk 2 above)
Comment (AlCutter) [inline cloud/google/util.sh]: I think all greps should do what I'm after here (or it has no business calling itself grep!)
Plus if I wanted the script to use my own 1337 grep I'd probably try shoving it in the `PATH` first.

**Pull request merged**